src/main/java/org/wso2/carbon/connector/operations/WriteFile.java

```diff
@@ -176,10 +176,10 @@ public void connect(MessageContext messageContext) throws ConnectException {
                 handleError(messageContext, e, Error.ILLEGAL_PATH, errorDetail);
 
             } catch (FileOperationException | IOException e) { //FileSystemException also handled here
-                log.error(e);
+                String errorDetail = ERROR_MESSAGE + targetFilePath;
+                log.error(errorDetail, e);
```

Comment:
will this solve the stack trace not showing the issue? log.error(e) didn't print the stack trace according to the logs

Reply:
Yes, In addition to that i have added logs for the connection factory as well via [c87b0fe](https://github.com/wso2-extensions/esb-connector-file/pull/213/commits/c87b0fe4dd64136166eb24d1b98f771594b6c549)